src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin square ($size) {
+  width: $day-side-length;
+  height: $day-side-length;
```

Comment:
you give in mixin variable $size, but use $day-side-length, problem is when I type like an example @include square(40px) it will not work at all. 
you need use in mixin exactly the same variable you declare = $size. it works like $size = variable that i transfer;
in your case it works only because you have $day-side-length declared

Reply:
yep!) will fix that